<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionX</class>
 <widget class="QWidget" name="OptionX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionX</string>
  </property>
  <widget class="QLineEdit" name="QtInitialPrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtInterestRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtVolatility">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="QtOptionType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>European Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>European Put</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>American Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>American Put</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Symmetric Power Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Symmetric Power Put</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asymmetric Power Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asymmetric Power Put</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Option Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>199</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Days Until Expiration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Initial Price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Volatility</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Strike Price</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="QtDaysUntilExpiration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtStrikePrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtPower">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtUpperBarrier">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtRebate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced Features</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Basic Features</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtCurrentToAveragePrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Interest Rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Upper barrier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Rebate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Current to Average Price (Asian)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>QtOptionType</tabstop>
  <tabstop>QtDaysUntilExpiration</tabstop>
  <tabstop>QtInitialPrice</tabstop>
  <tabstop>QtVolatility</tabstop>
  <tabstop>QtStrikePrice</tabstop>
  <tabstop>QtInterestRate</tabstop>
  <tabstop>QtPower</tabstop>
  <tabstop>QtUpperBarrier</tabstop>
  <tabstop>QtRebate</tabstop>
  <tabstop>QtCurrentToAveragePrice</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
